.github/workflows/ci-mssql.yml

```diff
@@ -65,42 +51,33 @@ jobs:
       - name: Install PHP with extensions
         uses: shivammathur/setup-php@v2
         with:
-          php-version: ${{ matrix.php }}
-          extensions: ${{ matrix.extensions }}
+          coverage: xdebug
+          extensions: ${{ env.EXTENSIONS }}
           ini-values: date.timezone='UTC'
+          php-version: ${{ matrix.php }}
           tools: composer:v2, pecl
 
-      - name: Determine composer cache directory on Linux
-        run: echo "COMPOSER_CACHE_DIR=$(composer config cache-dir)" >> $GITHUB_ENV
-
-      - name: Cache dependencies installed with composer
-        uses: actions/cache@v3
-        with:
-          path: ${{ env.COMPOSER_CACHE_DIR }}
-          key: php${{ matrix.php }}-composer-${{ hashFiles('**/composer.json') }}
-          restore-keys: |
-            php${{ matrix.php }}-composer-
-
       - name: Update composer
         run: composer self-update
 
       - name: Install dependencies with composer
         run: composer update --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
 
-      - name: Install dependencies with composer php 8.0
-        if: matrix.php == '8.0'
-        run: composer update --ignore-platform-reqs --prefer-dist --no-interaction --no-progress --optimize-autoloader --ansi
+      - name: Run MSSQL tests with PHPUnit and generate coverage.
+        if: matrix.php == '7.4'
+        run: vendor/bin/phpunit --group mssql --coverage-clover=coverage.xml --colors=always
 
-      - name: PHP Unit tests for PHP 7.1
-        run: vendor/bin/phpunit --coverage-clover=coverage.clover --group mssql --colors=always
-        if: matrix.php == '7.1'
+      - name: Run MSSQL tests with PHPUnit.
+        if: matrix.php == '8.0'
+        continue-on-error: true
```

Comment:
I'd remove all these since how it affects the overall status of the job: https://github.com/orgs/community/discussions/15452

Reply:
:)

Reply:
done.